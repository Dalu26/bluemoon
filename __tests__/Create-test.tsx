import 'react-native';
import React from 'react';
import CreateProduct from '../src/screens/more/createProduct';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const create = renderer.create(<CreateProduct />)
  expect(create).toMatchSnapshot();
});
