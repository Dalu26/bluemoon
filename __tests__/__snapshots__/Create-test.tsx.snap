// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "backgroundColor": "rgba(98, 47, 181, 1)",
      "flex": 1,
      "paddingTop": 53.5,
    }
  }
>
  <BVLinearGradient
    colors={
      Array [
        4284624821,
        4279963446,
      ]
    }
    endPoint={
      Object {
        "x": 0.5,
        "y": 1,
      }
    }
    locations={null}
    startPoint={
      Object {
        "x": 0.5,
        "y": 0,
      }
    }
    style={
      Object {
        "flex": 1,
        "marginTop": 23.5,
        "paddingHorizontal": 27,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "height": 79,
            "width": "100%",
          },
          Object {
            "display": "flex",
            "flexDirection": "row",
          },
        ]
      }
    >
      <View
        accessible={true}
        collapsable={false}
        focusable={true}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "#e1d5f5",
              "borderRadius": 9.5,
              "height": 55.5,
              "width": 67.5,
            },
            Object {
              "alignItems": "center",
              "display": "flex",
              "justifyContent": "center",
            },
          ]
        }
      >
        <Text
          allowFontScaling={false}
          selectable={false}
          style={
            Array [
              Object {
                "color": "#000",
                "fontSize": 20,
              },
              undefined,
              Object {
                "fontFamily": "Feather",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {},
            ]
          }
        >
          ï„¯
        </Text>
      </View>
    </View>
    <RCTScrollView
      keyboardShouldPersistTaps="always"
      showsVerticalScrollIndicator={false}
    >
      <View>
        <Text
          style={
            Array [
              Object {
                "color": "#ffffff",
                "fontFamily": "Poppins-Regular",
                "fontSize": 26,
              },
              Array [
                Object {
                  "fontFamily": "Poppins-Bold",
                },
                Object {
                  "color": "#ffffff",
                  "fontSize": 44,
                  "marginVertical": 31.5,
                },
              ],
            ]
          }
        >
          Add a new product to the inventory
        </Text>
        <View
          style={
            Array [
              Object {},
              undefined,
            ]
          }
        >
          <View
            style={
              Array [
                Object {
                  "display": "flex",
                  "flexDirection": "row",
                },
                Object {
                  "alignItems": "center",
                },
              ]
            }
          >
            <TextInput
              autoCapitalize="none"
              autoCorrect={false}
              autoFocus={true}
              maxLength={50}
              onBlur={[Function]}
              onChangeText={[Function]}
              onFocus={[Function]}
              placeholder="Name"
              placeholderTextColor="#ffffff"
              style={
                Array [
                  Object {
                    "backgroundColor": "#2B1A6C",
                    "borderRadius": 28.5,
                    "borderWidth": 2,
                    "color": "#ffffff",
                    "fontSize": 28,
                    "minHeight": 102.5,
                    "paddingHorizontal": 35,
                    "width": "100%",
                  },
                  Object {
                    "marginVertical": 8,
                  },
                  Object {
                    "fontFamily": "Poppins-Regular",
                  },
                  Object {
                    "borderColor": "transparent",
                  },
                ]
              }
              textAlignVertical="center"
              value=""
            />
          </View>
        </View>
        <View
          style={
            Array [
              Object {},
              undefined,
            ]
          }
        >
          <View
            style={
              Array [
                Object {
                  "display": "flex",
                  "flexDirection": "row",
                },
                Object {
                  "alignItems": "center",
                },
              ]
            }
          >
            <TextInput
              autoCapitalize="none"
              autoCorrect={false}
              keyboardType="numeric"
              maxLength={8}
              onBlur={[Function]}
              onChangeText={[Function]}
              onFocus={[Function]}
              placeholder="Price"
              placeholderTextColor="#ffffff"
              style={
                Array [
                  Object {
                    "backgroundColor": "#2B1A6C",
                    "borderRadius": 28.5,
                    "borderWidth": 2,
                    "color": "#ffffff",
                    "fontSize": 28,
                    "minHeight": 102.5,
                    "paddingHorizontal": 35,
                    "width": "100%",
                  },
                  Object {
                    "marginVertical": 8,
                  },
                  Object {
                    "fontFamily": "Poppins-Regular",
                  },
                  Object {
                    "borderColor": "transparent",
                  },
                ]
              }
              textAlignVertical="center"
              value=""
            />
          </View>
        </View>
        <View
          style={
            Array [
              Object {},
              undefined,
            ]
          }
        >
          <View
            style={
              Array [
                Object {
                  "display": "flex",
                  "flexDirection": "row",
                },
                Object {
                  "alignItems": "center",
                },
              ]
            }
          >
            <TextInput
              autoCapitalize="none"
              autoCorrect={false}
              keyboardType="numeric"
              maxLength={8}
              onBlur={[Function]}
              onChangeText={[Function]}
              onFocus={[Function]}
              placeholder="Total Stock"
              placeholderTextColor="#ffffff"
              style={
                Array [
                  Object {
                    "backgroundColor": "#2B1A6C",
                    "borderRadius": 28.5,
                    "borderWidth": 2,
                    "color": "#ffffff",
                    "fontSize": 28,
                    "minHeight": 102.5,
                    "paddingHorizontal": 35,
                    "width": "100%",
                  },
                  Object {
                    "marginVertical": 8,
                  },
                  Object {
                    "fontFamily": "Poppins-Regular",
                  },
                  Object {
                    "borderColor": "transparent",
                  },
                ]
              }
              textAlignVertical="center"
              value=""
            />
          </View>
        </View>
        <View
          style={
            Array [
              Object {},
              undefined,
            ]
          }
        >
          <View
            style={
              Array [
                Object {
                  "display": "flex",
                  "flexDirection": "row",
                },
                Object {
                  "alignItems": "center",
                },
              ]
            }
          >
            <TextInput
              autoCapitalize="none"
              autoCorrect={false}
              multiline={true}
              onBlur={[Function]}
              onChangeText={[Function]}
              onFocus={[Function]}
              placeholder="Description"
              placeholderTextColor="#ffffff"
              style={
                Array [
                  Object {
                    "backgroundColor": "#2B1A6C",
                    "borderRadius": 28.5,
                    "borderWidth": 2,
                    "color": "#ffffff",
                    "fontSize": 28,
                    "minHeight": 102.5,
                    "paddingHorizontal": 35,
                    "width": "100%",
                  },
                  Array [
                    Object {
                      "marginVertical": 8,
                    },
                    Object {
                      "height": 189.5,
                    },
                  ],
                  Object {
                    "fontFamily": "Poppins-Regular",
                  },
                  Object {
                    "borderColor": "transparent",
                  },
                ]
              }
              textAlignVertical="top"
              value=""
            />
          </View>
        </View>
        <View
          style={
            Object {
              "width": "100%",
            }
          }
        >
          <View
            accessibilityState={
              Object {
                "disabled": true,
              }
            }
            accessible={true}
            collapsable={false}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#7F4CD2",
                "borderRadius": 39.5,
                "display": "flex",
                "height": 155,
                "justifyContent": "center",
                "marginTop": 63,
                "opacity": 1,
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "display": "flex",
                  "flexDirection": "row",
                  "justifyContent": "center",
                }
              }
            >
              <Text
                style={
                  Array [
                    Object {
                      "color": "#ffffff",
                      "fontFamily": "Poppins-Regular",
                      "fontSize": 26,
                    },
                    Array [
                      Object {
                        "fontFamily": "Poppins-Medium",
                        "fontSize": 50.5,
                        "fontWeight": "600",
                        "lineHeight": 76,
                      },
                      undefined,
                      Object {
                        "color": "#AAAAB3",
                      },
                    ],
                  ]
                }
              >
                Add
              </Text>
            </View>
          </View>
        </View>
      </View>
    </RCTScrollView>
  </BVLinearGradient>
  <View
    style={
      Object {
        "alignItems": "center",
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 55.5,
      }
    }
  />
</RCTSafeAreaView>
`;
